src/web/crate_details.rs

```diff
@@ -174,11 +174,11 @@ impl CrateDetails {
 
         let metadata = MetaData {
             name: krate.name.clone(),
-            version: krate.version.clone(),
-            version_or_latest: version_or_latest.to_string(),
+            version: version.clone(),
+            req_version: req_version.unwrap_or_else(|| ReqVersion::Exact(version.clone())),
             description: krate.description.clone(),
             rustdoc_status: krate.rustdoc_status,
-            target_name: Some(krate.target_name.clone()),
+            package_name: Some(krate.target_name.clone()),
```

Comment:
This renaming is confusing, the `target_name` is the name of the library-target within the package, `name` is the name of the package itself. If anything this should be `library_name` since we only support libraries at the moment, but that would be wrong if we do get around to adding support for generating binary/example documentation.

Reply:
@Nemo157 you're probably right. I definitely confused the target-name more than once with the target _platform_ name. 

Let's move this discussion, I removed the commit from this PR, will create a separete PR with it.